pkg/jobrunaggregator/jobrunaggregatoranalyzer/analyzer_disruption.go

```diff
@@ -70,6 +70,9 @@ func (o *JobRunAggregatorAnalyzerOptions) CalculateDisruptionTestSuite(ctx conte
 				return nil, err
 			}
 
+			// we are correcting our disruption numbers, so we are forcing everything to skipped until 12/15
+			status = testCaseSkipped
```

Comment:
status can be reset to testCaseFailed just below here, should this be right before we return?

Reply:
n/m I misread == below